packages/core/src/getters/ref.ts

```diff
@@ -59,8 +58,18 @@ export const getRefInfo = (
     };
   }
 
+  const resolve = (from: string, to: string): string => {
```

Comment:
What do you think about moving it outside of the getRefInfo function? And rename it `resolveUrl`?